worldcup/src/components/random-result.jsx

```diff
@@ -0,0 +1,26 @@
+import { Question, Answer, AnswerImg, Button } from "../style/item";
+import { Stage } from "../style";
+import { RESULT_LIST } from "../assets/result-item";
+import Random from "./random";
+import { useState } from "react";
+
+function RandomResult(){
+    const [initial, setInitial] = useState(false);
+
+    const randomVal = Math.floor( Math.random()*RESULT_LIST.length );
+    const tempInfo = RESULT_LIST[randomVal];
+    const {link, description , ...indexs} = tempInfo;
+  
+    return initial === false 
+    ?(
+        <Stage>
+            <Question>크리스마스에 이거 하면 되겠다!</Question>
+            <AnswerImg src={link}></AnswerImg>
+            <Answer>🌟{description}❄️</Answer>
+            <Button onClick={ () => {setInitial(true)} }>다시하기</Button>
+        </Stage>
+    )
+    : <Random></Random>;  
```

Comment:
이렇게 내용물이 비어있는 열림/닫힘 태그를 이곳저곳에서 많이 사용해주었는데, 내부 children이 없는 대부분의 컴포넌트는 주로 단일태그를 사용한답니다 ! 
```suggestion
    : <Random />;  
```

Reply:
오 그렇군뇨! 사실 자동완성이 항상 저렇게 되길래 아무 생각 없이 썼는데 새로운 걸 알아갑니다❤
혹시 저렇게 단일 태그를 자동완성 시켜주는 별도의 extension이 있나요? 아님 제가 하나씩 쳐야되는 걸까요?? (물론, 후자가 아닐 거라고 생각은 합니다,,,)

Reply:
저는 항상 직접 작성해주곤 했었는데 가형이 코멘트 보고 찾아보니까 eslint-plugin-react에 self-closing-comp 라는 rule이 존재하네요!!! 대박대박 저도 몰랐는데 이거 꼭 써봐야겠어요 
[self-closing-comp rule](https://github.com/jsx-eslint/eslint-plugin-react/blob/master/docs/rules/self-closing-comp.md)

자료 참고하고, 앞으로 우리에게 닥칠 합세, 솝커톤, 앱잼에서 조원들과 **컨벤션**을 정하는 시간에 **eslint 규칙**도 정하게 될텐데, 그때 적용하자고 가형이가 의견 내보면 아주아주 좋을듯 !! ❤️

Reply:
참고로 얘는 셀프 클로징 태그 라고도 불러용-!